src/styles/less/map.less

```diff
@@ -0,0 +1,32 @@
+.googlemap {
+  height: 100vh;
+  width: 100%;
+  position: relative;
+}
+.autocomplete-search {
```

Comment:
Good work on getting the styling and autocomplete right for a unique ant design search box component.